### PR TITLE
Add an exampleConfig.js and documentation

### PR DESCRIPTION
**Pull request by esquivalient**
People that are new to StatsD (or alternative backends) often request help getting the configuration right. Also handy for setting up quick test environments.

### CODE REVIEW

File: exampleConfig.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+/*
+Example StatsD configuration for sending metrics to Instrumental.
+See the offical Statsd exampleConfig.js for more StatsD options.
+*/
+{
+ port: 8125
+, backends: ["statsd-instrumental-backend" ]
+, debug: false
+, instrumental: {
+    key: "INSTRUMENTAL_API_KEY", // REQUIRED
```
Comment (JamesPaden) [inline exampleConfig.js]: I think we should be specific at project vs user token in these examples. Also, I think we've standardized on the term "token" through UX and upgraded docs. So `PROKECT_API_TOKEN`?

**Comment by jqr** [comment]
:+1:

File: exampleConfig.js (hunk 1 above)
Comment (janxious) [inline exampleConfig.js]: `PROJECT_API_TOKEN` is correct here.

**Comment by mediocretes** [comment]
:up:

**Pull request merged**